### PR TITLE
"No Rights" results in an "invalid" image

### PR DESCRIPTION
**Pull request by kenoir**
In order to prevent use of "No Rights" images and to encourage entering better rights data on upload.

**Image page**
imagep_inv_c

**Search result**
search_inv

**Upload page**
upload_inv

### CODE REVIEW

**Comment by paperboyo** [comment]
Worth making invalid more obvious/different from paid? E.g.

invalid_obvious
[added (ugly!) sign and a red frame around UR, haven't "updated" the tooltip]

Possible downside of having an image overlay would be making harder a use case of using a Grid as a temporary selection machine (I will upload these and decide which one(s) to use later...). For users that do that (or using lowres positionals) an easy way to search just for their invalid files (in order to delete them and not pollute the system), would be nice to have.

**Comment by kenoir** [comment]
@paperboyo 
norights

**Comment by paperboyo** [comment]
Mine and yours are equally bad. Maybe it should just be a pic of Nicholas (instead of Arnold)? I trust @akemitakagi.

File: kahuna/public/js/components/gr-metadata-validity/gr-metadata-validity.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div ng:if="! image.data.valid"
     class="image-notice image-info__group validity validity--invalid validity--invalid--point-up">
     <strong>Incomplete metadata</strong>
-    Fill in description and credit before use
+    Fill in description, credit and usable rights
```
Comment (jamesgorrie) [inline kahuna/public/js/components/gr-metadata-validity/gr-metadata-validity.html]: "usable"? Usage?

Comment (kenoir) [inline kahuna/public/js/components/gr-metadata-validity/gr-metadata-validity.html]: "rights that make this image usable"

I am not good at this.

**Comment by NickPapacostas** [comment]
👍 🉐 

one small point, if you change the rights to "no rights" on a pic that was previously croppable, the "can't crop" error only comes up after you try cropping (as opposed to the button saying "cannot crop")... 

just fyi, dont think it matters.

**Comment by kenoir** [comment]
@NickPapacostas Needed to watch changes and update deliberately.

https://github.com/guardian/grid/pull/1828/commits/b6f9c10d8b3c876367d3f2cbba42807d7cde5887

**Comment by NickPapacostas** [comment]
👍

**Pull request merged**